bofire/data_models/outlier_detection/outlier_detections.py

```diff
@@ -0,0 +1,53 @@
+import itertools
+from typing import List
+
+from bofire.data_models.base import BaseModel
+from bofire.data_models.domain.api import Inputs, Outputs
+from bofire.data_models.outlier_detection.outlier_detection import IterativeTrimming
+
+AnyOutlierDetector = IterativeTrimming
+
+
+class OutlierDetections(BaseModel):
+    """ "List of Outlier detectors.
+
+    Behaves similar to a outlier_detector."""
+
+    detectors: List[AnyOutlierDetector]
```

Comment:
```suggestion
    detectors: Annotated[List[AnyOutlierDetector], Field(min_items=1]
```
This ensure, that the detectors list is not empty, we should also test that the `OutlierDetectors` functional model  can work with just one detector.